psydac/api/ast/nodes.py

```diff
@@ -294,9 +329,41 @@ class RAT(Basic):
 class EvalMapping(BaseNode):
     """
         This function computes atomic expressions needed
-        to evaluate  EvalMapping final expression
+        to evaluate  EvalMapping final expression.
+
+        Parameters:
+        ----------
+        quads: <IndexQuadrature>
+            Indices used for the quadrature loops
+
+        indices_basis : <IndexDofTest>
+            Indices used for the basis loops
+
+        q_basis : <GlobalTensorQuadratureTestBasis>
+            The 1d basis function of the tensor-product space
+
+        mapping : <Mapping>
+            Sympde Mapping object
+
+        components  : <GeometryExpressions>
+            The 1d coefficients of the mapping
+
+        mapping_space : <VectorSpace>
+            The vector space of the mapping
+
+        tests   : tuple_like (Variable)
+            The dummy variable for the test functions
+
+        nderiv  : <int>
+            Maximum number of derivatives
+
+        mask    : int,optional
+            The fixed direction in case of a boundary integral
+
+        is_irational: bool,optional
+            True if the mapping is irational
```

Comment:
Same typos on both lines: `irational` -> `rational` (without the leading `i`)

Reply:
lol my bad

Reply:
done